### ISSUE TITLE
Using autocomplete with form field

### ISSUE THREAD

**Issue by pfremp** [yourlabs/django-autocomplete-light#467]
Hello, I'm trying to use autocomplete in my form. I have installed everything and I'm trying to test it by using the examples in the docs. 

Please see snippets of my code below:

autocomplete_light_registry.py

import autocomplete_light

class OsAutocomplete(autocomplete_light.AutocompleteListBase):
       choices = ['Linux', 'BSD', 'Minix']

autocomplete_light.register(OsAutocomplete)

forms.py

import autocomplete_light

class PartDetailsForm (forms.ModelForm):
    region = autocomplete_light.ChoiceField('OsAutocomplete')
    city = forms.CharField(required=False, label='City', max_length=128)
    ....

In my base.html
<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.9.1/jquery.js" type="text/javascript"></script>
{% include 'autocomplete_light/static.html' %}

In my urls.py I have 
....
url(r'^autocomplete/', include('autocomplete_light.urls')),

I'm new to django and python so not too sure if it is something simple I am overlooking. Ultimately I would like to use autocomplete in conjunction with cities_light in order to allow the user to enter their city and country in the form. 

Any advice is appreciated. 
![screen shot 2015-07-20 at 02 56 17](https://cloud.githubusercontent.com/assets/6531222/8769043/c6e856be-2e8a-11e5-9037-21a28fde41f1.png)

**Comment by jpic** [comment]
So what's the problem here exactly ?

**Comment by jpic** [comment]
BTW here's an [example with cities_light to tie 2 autocompletes](http://django-autocomplete-light.readthedocs.org/en/latest/dependant.html) which might look like what you'd want to do with region and city fields.

**Comment by pfremp** [comment]
The problem is when I being to type in the "Region" field, nothing happens. I'm not too sure if the JavaScript has been configured properly. 

In the example you posted above, what file should the JavaScript be pasted into. I'm at the "JavaScript" step?

Thanks again!

**Comment by jpic** [comment]
It doesn't matter where you put the JS, as long as it is loaded.

If nothing happens when you type something in the region field, maybe the js is not properly loaded (autocomplete.js, widget.js) or maybe the extensions it installed are overridden (ie. jquery is being loaded again in your page).

Anyway, I recommend that you try to reproduce this problem in a sample project.

**Comment by pfremp** [comment]
Thank you, it was the jquery being loaded multiple times that caused the issue. Seems to be working now. 

With regards to the styling of the form, is there any way to edit this? As you can see in the pic below, the autocomplete fields are displayed different from the standard fields.

![screen shot 2015-07-20 at 22 21 24](https://cloud.githubusercontent.com/assets/6531222/8787754/c6a5bc02-2f2d-11e5-866e-bca98a88627b.png)

Many thanks

**Comment by jpic** [comment]
I'm sorry but we support django admin which relies on overflow: hidden
instead of the clearfix technique to clear floats and it makes it a bit
harder to design.

Pull requests are welcome, CSS is not my area of expertise but I'd love to
ear you out.

**Comment by pfremp** [comment]
I created a separate stylesheet to override the default style:

label {
  display: block;
  max-width: 100%;
  margin-bottom: 5px;
  font-weight: bold;
}

.autocomplete-light-widget {
    display: block;
}

![screen shot 2015-07-20 at 23 23 04](https://cloud.githubusercontent.com/assets/6531222/8788931/13d166fe-2f36-11e5-86bf-b9d32bd96b42.png)

**Comment by jpic** [comment]
Looks neat B) anything else ?

**Comment by pfremp** [comment]
That's all for now. Thanks very much for your support and patience.

**Comment by jpic** [comment]
NP, enjoy
